internal/controller/controller.go

```diff
@@ -239,3 +251,87 @@ func (controller *Controller) SSHAddress() (string, bool) {
 
 	return controller.sshServer.Address(), true
 }
+
+func (controller *Controller) initializeMetrics() error {
```

Comment:
I think having online/offline workers counter is also beneficial.

Reply:
Implemented in https://github.com/cirruslabs/orchard/pull/185/commits/ea0e1f70eb843fc146b6b92a074a8f4dde6e1221.